docs/csharp/properties.md

```diff
@@ -54,6 +54,29 @@ public class Person
 The syntax shown above is the *auto property* syntax. The compiler
 generates the storage location for the field that backs up the 
 property. The compiler also implements the body of the `get` and `set` accessors.
+
+Sometimes, you need to initialize a property to
+a value other than the default for its type.  C# enables
+that by setting a value after the closing brace for the
+property. You may prefer the initial value for the `FirstName`
+property to be the emtpy string rather than `null`. You would
+specify that as shown below:
+
+```csharp
+public class Person
+{
+    public string FirstName
+    {
+        get;
+        set;
+    } = string.Empty;
```

Comment:
I think that for auto-properties, it's more common to format them on a single line. It looks especially weird to me because of the initializer. Though if that was changed here, other auto-properties on this page should probably be changed too.

Also, the comment below is indented by one tab, while the property is indented by four spaces. That makes it look inconsistently indented, at least on GitHub. The new code examples below have similar issues.